sbt/src/sbt-test/source-dependencies/pipelining/test

```diff
@@ -0,0 +1,11 @@
+> dep/compile
+
+> use/checkPickle
+$ sleep 1000
```

Comment:
What is the purpose of the sleep?

Reply:
We time wrap the stamp, and some of the platforms do not have the system time resolution, so I think I've put that in there to make sure things validate or invalidate.

Reply:
Right. So maybe we could set the modified time manually instead of sleeping?